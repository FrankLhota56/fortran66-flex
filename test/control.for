C
C PAUSE / STOP checks
C23456
      Pause 0123
      ST OP
      END
