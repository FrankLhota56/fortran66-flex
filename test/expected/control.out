Line     4: PAUSE
Line     4: OCTAL(00123)
Line     5: EOS
Line     5: STOP
Line     6: EOS
Line     6: END
